src/Sentry/Protocol/Envelopes/Envelope.cs

```diff
@@ -320,6 +321,18 @@ public static Envelope FromTransaction(Transaction transaction)
         return new Envelope(eventId, header, items);
     }
 
+    /// <summary>
+    /// Creates an envelope that contains one or more Metrics
+    /// </summary>
+    internal static Envelope FromMetrics(IEnumerable<Metric> metrics)
```

Comment:
if all we do with this is `ToArray` maybe we should take an array here instead of IEnumerable?